data_adapter/preprocessing.py

```diff
@@ -315,7 +315,7 @@ def __apply_parameter_merge(self, data, datatype):
 
         series = pd.Series(dtype=object)
         for column in data.columns:
-            if column in ["id", *groups]:
+            if column in ["id", "version", *groups]:
```

Comment:
Is there still a check if version matches within scalar / FK table?

Reply:
No check there. IMO this is not necessary. As versions within one dataset should be equal...